app/controllers/simple_discussion/application_controller.rb

```diff
@@ -12,7 +12,7 @@ def is_moderator_or_owner?(object)
   end
   helper_method :is_moderator_or_owner?
 
-  def is_moderator?
+  def self.is_moderator?
```

Comment:
Shouldn't this work without the `self`? The controllers inherit from ApplicationController so this method should be available as normal inside ForumPostsController.